types/experiments.ts

```diff
@@ -162,3 +168,11 @@ interface Branch {
 
   feature?: FeatureConfig;
 }
+
+interface Outcome {
+  /** Identifier for the outcome */
+  slug: string;
+
+  /** e.g. "primary" or "secondary" */
+  priority: string;
```

Comment:
Would it make sense to be more explicit about the values allowed here?
```suggestion
  priority: "primary" | "secondary";
```

Reply:
You know, actually, dumb question: will e.g. the Rust SDK give up and die if it sees a value it doesn't know about here? I'm not sure if we've formalized an idea of a safe schema change.

I hesitated to do this because I didn't want messages to fail to validate if another value appeared but maybe that's the wrong way to think about it.

Reply:
Yes, it will ignore additional fields

Reply:
I'd go for `string` since we generally enforce this at the experimenter level (using specific values here makes it harder to extend for clients on older versions in the SDK) but de facto it won't matter since this isn't used by the SDK